plans/create.pp

```diff
@@ -74,10 +80,38 @@
   Optional[String[1]] $environment               = 'production',
   Optional[Enum['linux', 'windows']] $os_type    = 'linux',
   Optional[Provision::ProviderOptions] $provider_options = undef,
+
+  ##GCP Specific Parameters
```

Comment:
except `project` others can goes into `provider_options` ?

Reply:
Sure, Implemented the suggestion.